components/avatar.css

```diff
@@ -0,0 +1,54 @@
+.avatar__example-demo-container{
```

Comment:
This class can be given in css file of documentation as it is project specific.

Reply:
Nice catch there. Thanks for that. Refactored and pushed.